sprint/start_clean.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+#Clear terminal screen so the about text can be read
+clear
+
+RED='\033[31m'
+GREEN='\033[32m'
+YELLOW='\033[33m'
+RESET='\033[0m'
+
+printf "
+${GREEN}
+####
+# This simple script starts a clean instance of drupal 
+# running in ddev and imports a starter database.
+#
+# Make sure you've uploaded any patchs from last issue 
+# you worked on before continuing.
+#
+# Press y to continue
+# !!You don't need to hit enter!!.
+#
+####
+${RESET}"
+read -n1 INSTALL
+if [[ ! $INSTALL =~ ^[Yy]$ ]]
+then
+    exit 1
+fi
+
+ddev remove
+ddev start
+wait
+ddev exec git fetch
+ddev exec git reset --hard origin/8.6.x
+ddev exec composer install
+ddev exec drush si standard --db-url=mysql://db:db@db:3306/db
```

Comment:
Follow-up for this to add `--account-pass`?

Reply:
Have added this now